pacta/README.md

```diff
@@ -1,91 +1,122 @@
 # Description
 
-The Dockerfile in this directory creates an image containing a freshly cloned
-copies of PACTA_analysis and all the repositories it depends on.
+The Dockerfile in this directory creates an image containing a freshly
+cloned copies of PACTA_analysis and all the repositories it depends on.
 
 The tree of the docker container looks like this:
 
-```bash
+``` {.bash}
 /bound  # contents of PACTA_analysis
 /pacta-data
 /create_interactive_report
 # ... more siblings of PACTA_analysis
 ```
 
-# Usage
+# Notes
 
 Note that PACTA_analysis and friends are not mounted but copied into the
 container, so they will be frozen in the state they are when you build
 the image.
 
 # Usage
 
-Run the build_with_tag.sh script, specifying a tag to assign
-to it. Ideally the tag should use semantic versioning, and should follow
-previously existing tags in the PACTA_analysis and friends repos.
-
-```bash
+Before running the script, you will need to remove any existing
+2dii_pacta docker images that are loaded. You can do that, for instance,
+with `docker rmi 2dii_pacta`.
+
+Also before running the script, you will need to choose the tag that you
+want to use for the release. You should use [semantic
+versioning](https://semver.org), and you should choose a tag that
+follows in sequence from previously existing tags in the PACTA_analysis 
+and friends repos. You can see existing tags in the relevant repos
+here:\
+<https://github.com/2DegreesInvesting/pacta-data/tags>\
+<https://github.com/2DegreesInvesting/create_interactive_report/tags>\
+<https://github.com/2DegreesInvesting/StressTestingModelDev/tags>\
+<https://github.com/2DegreesInvesting/pacta-data/tags>\
+<https://github.com/2DegreesInvesting/docker/tags>
+
+Run the build_with_tag.sh script, specifying a tag to assign to it.
+
+``` {.bash}
 ./build_with_tag.sh 0.0.4
 ```
 
 The script will:
 
-- clone the repos locally, only copying the current version of the files
-- remove any existing "2dii_pacta" images from your local docker system
-- build a "2dii_pacta:<tag>" and "2dii_pacta:latest" docker image (where <tag>
-is the tag you provided, e.g. 0.0.4). The image builds from the Dockerfile
-in this directory, which will
-  - use 2dii/r-packages as a base
-  - copy in the freshly cloned repos
-  - make some necessary permissions changes
-  - export the freshly made docker image gzipped (2dii_pacta.tar.gz)
+-   clone the repos locally, only copying the current version of the
+    files
```

Comment:
Is the white space between '-" and the test intentional?

Reply:
nope... RStudio did that too... got rid of it in a new commit

Reply:
Interesting. I'm generally interested in automated refactoring tools, and also interested in how RStudio drives people's style.
How did that happen? Did you click any formatting option in particular?